script/hipify-inplace.sh

```diff
@@ -51,27 +60,47 @@ echo "Logs written to ${HIPIFY_LOG}"
 
 # Additional fixes for project-specific things and things hipify misses or gets
 # wrong.
-for src in ${srcs[@]}; do
-    sed -i 's@#include <hipblas.h>@#include <hipblas/hipblas.h>@' $src
-    sed -i 's@#include <hipsparse.h>@#include <hipsparse/hipsparse.h>@' $src
-    sed -i 's@#include <cuda_fp8.h>@#include <hip/hip_fp8.h>@' $src
-    sed -i 's@#include <cuda_bf16.h>@#include <hip/hip_bf16.h>@' $src
-    sed -i 's@\bDGL_USE_CUDA\b@DGL_USE_ROCM@g' $src
-    sed -i 's@\bCUB_VERSION\b@HIPCUB_VERSION@g' $src
-    sed -i 's@\bCUDART_ZERO_BF16\b@HIPRT_ZERO_BF16@g' $src
-    sed -i 's@\bCUDART_INF_BF16\b@HIPRT_INF_BF16@g' $src
-    sed -i 's@\bthrust::cuda::par@thrust::hip::par@g' $src
-    sed -i 's@\b__nv_fp8_e4m3\b@__hip_fp8_e4m3@g' $src
-    sed -i 's@\b__nv_fp8_e5m2\b@__hip_fp8_e5m2@g' $src
-    sed -i 's@\bCUBLAS_GEMM_DEFAULT_TENSOR_OP\b@HIPBLAS_GEMM_DEFAULT@g' $src
-    sed -i 's@\bcurand4\b@hiprand4@g' $src
-    # hipify uses the old one
-    sed -i 's@\bhip_bfloat16\b@__hip_bfloat16@g' $src
-    sed -i 's@\b__trap();@abort();@' $src
+for src in ${all_srcs[@]}; do
+    sed -i -e 's@#include <hipblas.h>@#include <hipblas/hipblas.h>@' \
+        -e 's@#include <hipsparse.h>@#include <hipsparse/hipsparse.h>@' \
+        -e 's@#include <cuda_fp8.h>@#include <hip/hip_fp8.h>@' \
+        -e 's@#include <cuda_bf16.h>@#include <hip/hip_bf16.h>@' \
+	-e 's@#include <cub/cub.cuh>@#include <hipcub/hipcub.hpp>@' `# TODO: File issue` \
```

Comment:
EARGH tabs!?!?!?

Reply:
I have to admit, I have gotten lazy about copying a vim config to new files :D

I'll be formatting all of these commits today.

Reply:
All tabs should be removed now and I've run clang-format now.